### ISSUE TITLE
Cannot read properties of null (reading 'name')

### ISSUE THREAD

**Issue by josuanbn** [VemtoOrg/vemto2-issues#105]
**Error Message:**
Cannot read properties of null (reading 'name')

**Stack:**
```
TypeError: Cannot read properties of null (reading 'name')
    at Hee.hasDifferentPivot (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:110:25535)
    at Hee.needsToAddPivotToModelTemplate (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:110:25400)
    at Object.eval (eval at compileAsync (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:315:649), <anonymous>:304:39)
    at Dae.compileAsync (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:315:675)
    at Dae.compileAsyncWithErrorTreatment (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:315:290)
    at Vy.compile (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:361:14378)
    at Vy.compileWithImports (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:361:13975)
    at async IR.compile (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:363:4479)
    at async IR.compileWithErrorThreatment (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:363:3600)
    at async xi (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:1261:14426)
```

**Template Error Line:**
```
275
```

**Template Section:**
```

```

**How to reproduce:**
1. 
2. 
3. 

**Expected behavior:**

**Actual behavior:**

**Additional context:**

---

Vemto 2.0.8
win32 10.0.17763
Locale: es
App version: 2.0.8

**Comment by josuanbn** [comment]
Hi, this error appears after modifying a relationship. The only solution I found is delete the table/model and then  recreate the table/model and the new relationship

**Comment by TiagoSilvaPereira** [comment]
@josuanbn thanks for reporting, I'll check what is happening

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.10
